.docker/clickhouse/single_node/config.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0"?>
```

Comment:
It would be nice to have a separate PR or follow [conventionalcommits](https://www.conventionalcommits.org/en/v1.0.0/). I should squash commits now, and two unrelated changes will be merged.

Reply:
The reason why I added this is because I was not sure if the query_log would work by default without a config. I will check. If it works, I will create a separate PR instead.

Reply:
Looks like the query_log is (now?) enabled by default. I will make a separate one.